tests/universe/InstallPackageModal-cy.js

```diff
@@ -14,30 +14,8 @@ describe('Install Package Modal', function () {
 
   it('displays install modal for package', function () {
     cy
-      .get('.modal .form-element-inline-text')
-      .should('contain', 'marathon-user');
-  });
-
-  it('displays config header for package', function () {
```

Comment:
In stead of just removing this, could we move them to test the same thing in the advanced modal?